dash/dash.py

```diff
@@ -258,6 +261,10 @@ class Dash(object):
     Set to None or '' if you don't want the document.title to change or if you
     want to control the document.title through a separate component or
     clientside callback.
+
+    :param long_callback_manager: Long callback manager instance to support the
+    ``@app.long_callback`` decorator. Currently one of
+    ``DiskcacheLongCallbackManager`` or ``CeleryLongCallbackManager``
```

Comment:
Even though the first sentence should be clear, the second sentence makes it sound like you can just give the class, like `long_callback_manager=DiskcacheLongCallbackManager`

Reply:
Good point. updated in 3892ecb (apologies for the bogus commit message)